roles/setup/tasks/setup-local.yml

```diff
@@ -68,6 +68,14 @@
     group: root
     mode: 'u=rwx,go=rx'
 
+- name: Create receptor's log folder
+  ansible.builtin.fole:
+    path: "{{ receptor_path_log }}"
+    state: directory
```

Comment:
this is creating a directory but your variable is the full log path, so this is leading to an error.

I'm okay with either choice -- either provide the full log path or just the directory